src/resolvers.ts

```diff
@@ -4,12 +4,20 @@ import { User } from './entity/User';
 import * as bcrypt from 'bcrypt';
 import { InputError, UnauthorizedError } from './test/error';
 import Jwt from 'jsonwebtoken';
-import { CreateUserInput, LoginInput } from './schema';
+import { CreateUserInput, IDInput, LoginInput } from './schema';
 import { verifyJWT } from './verify';
 
 export const resolvers = {
   Query: {
     hello: () => 'Hello World',
+    user: async (_treco: any, args: IDInput, ctx: any) => {
```

Comment:
melhor nao colocar qqr coisa nos codigos haha
tudo bem que aqui é só onboarding, mas é bom já sempre se atentar que em projetos para clientes, às vezes eles podem ter acesso ao que escrevemos e pode ficar ruim ter palavras aleatorias no meio do projeto kkk
```suggestion
    user: async (_: any, args: IDInput, ctx: any) => {
```

Reply:
e poderia criar uma interface para o context, ne? Para que nao precise tipar como `any` sempre 🤔